### PR TITLE
Docs: FAQ updates

### PR DESCRIPTION
**Pull request by brianshumate**
- Add a FAQ for per-key value size limitation in the KV
- Update Atlas FAQ with post-deprecation recommendations

### CODE REVIEW

File: website/source/docs/faq.html.markdown [hunk 1]
```diff
@@ -101,3 +101,12 @@ connect to a single server and so preparation for this possibility is helpful.
...
 scrutinize your own environment's specific needs and identify the root cause
 of any excessive resource utilization before arbitrarily increasing the limits.
+
+## Q: What is the per-key value size limitation for Consul's key/value store?
+
+The limit on a key's value size is 512KB. This is is strictly enforced and an
+HTTP 413 status will be returned to any client that attempts to store more
+than that limit in a value. It should be noted that the Consul key/value store
+is not designed to be used as a general purpose database. See
+[Server Performance](https://www.consul.io/docs/guides/performance.html) for
```
Comment (slackpad) [inline website/source/docs/faq.html.markdown]: this can just be `/docs/guides/performance.html`

File: website/source/docs/faq.html.markdown [hunk 2]
```diff
@@ -20,18 +20,18 @@ and [`disable_update_check`](/docs/agent/options.html#disable_update_check).
...
-all connected agents (data acquisition).
+Hosted Consul Enterprise in Atlas was officially deprecated on March 7th,
+2017.
 
-Standard ACLs can be applied to the SCADA connection, ensuring that Atlas is given only
-those privileges that make sense for your deployment.
+There are strong alternatives available and they are listed below.
 
-Using the SCADA service is optional. SCADA is only enabled by opt-in.
+For users on AWS the [-retry-join-ec2 configuration options](https://www.consul.io/docs/agent/options.html#_retry_join_ec2_tag_key) allow bootstrapping by automatically discovering AWS instances with a given tag key/value at startup. For users not on AWS the native [-join and retry-join functionality](https://www.consul.io/docs/agent/options.html#_join) can be used.
```
Comment (slackpad) [inline website/source/docs/faq.html.markdown]: Can you add pointers to the GCE stuff as well? These links can also get the `https://www.consul.io` stripped off the front.

File: website/source/docs/faq.html.markdown (hunk 1 above)
Comment (brianshumate) [inline website/source/docs/faq.html.markdown]: Thanks, fixed!

File: website/source/docs/faq.html.markdown (hunk 2 above)
Comment (brianshumate) [inline website/source/docs/faq.html.markdown]: Added GCE and fixed links.

**Comment by slackpad** [comment]
Thanks!

**Pull request merged**